src/analytics/userProperties.ts

```diff
@@ -23,8 +23,8 @@ export interface UserProperties {
   // nft offers
   nftOffersAmount?: number;
   nftOffersUSDValue?: number;
-  nftOffersMeanOfferVariance?: number;
-  nftOffersMedianOfferVariance?: number;
+  nftOffersMedianFloorDifferencePercentage?: number;
```

Comment:
not sure if its worth switching these values cause itll mess with the reports we currently have